### DISCUSSION TITLE
error type on $route.params.id in dynamic routes

### DISCUSSION THREAD

**Discussion by AtharFazle** [posva/unplugin-vue-router#461] [Questions and Help]
![Screenshot 2024-07-31 165249](https://github.com/user-attachments/assets/06f103cb-03a7-484b-bcc8-cb6507120d0f)

Property 'id' does not exist on type 'Record<never, never> | { path: string; } | { id: string; }'.
Property 'id' does not exist on type 'Record<never, never>'.

**Comment by posva** [comment]
Duplicate of #462
